src/inspector.rs

```diff
@@ -925,6 +950,58 @@ impl V8Inspector {
       )
     }
   }
+
+  #[allow(clippy::too_many_arguments)]
+  pub fn exception_thrown(
+    &mut self,
+    context: Local<Context>,
+    message: StringView,
+    exception: Local<Value>,
+    detailed_message: StringView,
+    url: StringView,
+    line_number: u32,
+    column_number: u32,
+    stack_trace: UniqueRef<V8StackTrace>,
+    script_id: i32,
+  ) -> u32 {
+    unsafe {
+      v8_inspector__V8Inspector__exceptionThrown(
+        self,
+        &*context,
+        message,
+        &*exception,
+        detailed_message,
+        url,
+        line_number,
+        column_number,
+        stack_trace.into_raw(),
+        script_id,
+      )
+    }
+  }
+
+  pub fn capture_stack_trace(
+    &mut self,
+    full_stack: bool,
+  ) -> UniqueRef<V8StackTrace> {
+    unsafe {
+      UniqueRef::from_raw(v8_inspector__V8Inspector__captureStackTrace(
+        self, full_stack,
+      ))
+    }
+  }
+
+  pub fn create_stack_trace(
+    &mut self,
+    stack_trace: Local<StackTrace>,
+  ) -> UniqueRef<V8StackTrace> {
+    unsafe {
+      UniqueRef::from_raw(v8_inspector__V8Inspector__createStackTrace(
```

Comment:
In C++ `V8Inspector::createStackTrace` is actually returning `nullptr` it seems. 
You weren't catching that in a previous version of this patch because you were using `NonNull::new_unchecked()` so the null value wasn't caught.

Reply:
I see, but it only partially makes sense to me - after all in the test I'm getting a stack trace but it has zero frames. So in this case shouldn't we actually use `UniquePtr<V8StackTrace>` which can be `null`?

Reply:
> I see, but it only partially makes sense to me - after all in the test I'm getting a stack trace but it has zero frames. So in this case shouldn't we actually use `UniquePtr<V8StackTrace>` which can be `null`?

Yes it looks like we should be using `UniquePtr<V8StackTrace>`.